src/themes/five/src/components/OrderDetails/index.js

```diff
@@ -574,7 +574,7 @@ const OrderDetailsUI = (props) => {
                           style={{ fontSize: 14 }}
                           color={order?.status === 20 ? 'secundary' : 'primary'}
                           onClick={() => handleChangeOrderStatus(20)}
-                          disabled={order?.status === 20 || order?.status === 21}
+                          disabled={order?.status === 20 || order?.status === 21 || order?.status === 15 || order?.status === 1}
```

Comment:
Add a constant with the keys 

const statusToDisable = [1, 15, 20, 21]

and then you can used it

disbaled={statusToDisable.includes(order?.status)}